client/components/theme/index.jsx

```diff
@@ -38,6 +38,8 @@ const Theme = React.createClass( {
 		active: React.PropTypes.bool,
 		// If true, the user has 'purchased' the theme
 		purchased: React.PropTypes.bool,
+		// If true, the theme is being installed (on a Jetpack site)
```

Comment:
Small nit. I would drop the ` (on a Jetpack site)`  for `Theme` this is irrelevant.